src/gui/touchscreengui.h

```diff
@@ -204,12 +202,13 @@ class TouchScreenGUI
 	IrrlichtDevice *m_device;
 	IGUIEnvironment *m_guienv;
 	IEventReceiver *m_receiver;
-	ISimpleTextureSource *m_texturesource;
+	ISimpleTextureSource *m_texturesource{};
 	v2u32 m_screensize;
+	s32 button_size;
 	double m_touchscreen_threshold;
 	std::map<int, rect<s32>> m_hud_rects;
-	std::map<int, irr::EKEY_CODE> m_hud_ids;
-	bool m_visible; // is the gui visible
+	std::map<size_t, irr::EKEY_CODE> m_hud_ids;
+	bool m_visible{}; // is the gui visible
```

Comment:
For `m_texturesource`, `m_visible` and `m_key_events`: do you intend to fix any certain bug, or why are those brackets added?